build.sbt

```
@@ -14,6 +14,16 @@ val commonSettings: Seq[Setting[_]] = Seq(
   organization := "org.scala-js",
   scalacOptions ++= Seq("-deprecation", "-feature", "-Xfatal-warnings"),
 
+  scalacOptions ++= {
+    if (isSnapshot.value)
+      Seq.empty
+    else {
+      val a = baseDirectory.value.toURI
```

`baseDirectory` depends on the project, whereas the URL is the same for all projects, so this will only work for the root project. Either move those options to the root project only (which is fine because that's the only project that gets published) or apply the following change, so that it would work in all projects:
```suggestion
      val a = (baseDirectory in LocalRootProject).value.toURI
```